tty/usbacm/usbacm.c

```diff
@@ -590,19 +713,13 @@ int main(int argc, char *argv[])
 	usbacm_common.lastId = 1;
 
 	for (i = 0; i < USBACM_N_MSG_THREADS; i++) {
-		if ((ret = beginthread(usbacm_msgthr, 4, usbacm_common.stack[i], sizeof(usbacm_common.stack[i]), NULL)) != 0) {
-			fprintf(stderr, "usbacm: fail to beginthread ret: %d\n", ret);
-			return 1;
-		}
-	}
-
-	for (i = 0; i < USBACM_N_UMSG_THREADS; i++) {
-		if ((ret = beginthread(usbthr, 3, usbacm_common.ustack[i], sizeof(usbacm_common.ustack[i]), NULL)) != 0) {
+		if ((ret = beginthread(usbacm_msgthr, 3, usbacm_common.stack[i], sizeof(usbacm_common.stack[i]), NULL)) != 0) {
```

Comment:
right now beginthread does indeed return 0, but maybe we'll extend it's API to return TID on success instead
```suggestion
		ret = beginthread(usbacm_msgthr, 3, usbacm_common.stack[i], sizeof(usbacm_common.stack[i]), NULL);
		if (ret < 0) {
```